charts/loki-distributed/values.yaml

```diff
@@ -76,6 +76,9 @@ loki:
     allowPrivilegeEscalation: false
   # -- Specify an existing secret containing loki configuration. If non-empty, overrides `loki.config`
   existingSecretForConfig: ""
+  # -- When 'existingSecretForConfig' is not empty, then changing 'externalConfigVersion' triggers restart of services -
+  # otherwise changes to the configuration cause a restart.
+  externalConfigVersion: '0'
```

Comment:
💭 None of these variables seem to be documented in the README.md for loki-distributed. Therefore I left out docs that explain the addition of this new `.Values` variable.